### PR TITLE
UIORGS-35 project setup

### PR DESCRIPTION
**Pull request by NikitaSedyx**
https://issues.folio.org/browse/UIORGS-35

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "parser": "babel-eslint",
+  "extends": "@folio/eslint-config-stripes"
+}
```
Comment (aliaksei-chumakou) [inline .eslintrc]: Should we use eslint config from ui-orders?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  },
+  "dependencies": {
+    "@folio/organizations": "^1.0.0",
+    "classnames": "^2.2.5",
+    "lodash": "^4.17.11",
+    "prop-types": "^15.6.0",
+    "react-intl": "^2.8.0"
+  },
+  "peerDependencies": {
+    "@folio/stripes": "^2.0.0",
```
Comment (aliaksei-chumakou) [inline package.json]: could you please change it to ^2.6.0 ? thanks

File: package.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    "test": "stripes test karma"
+  },
+  "devDependencies": {
+    "@bigtest/cli": "^0.2.2",
+    "@bigtest/interactor": "^0.9.2",
+    "@bigtest/mirage": "^0.0.1",
+    "@bigtest/mocha": "^0.5.2",
+    "@bigtest/react": "^0.1.2",
+    "@folio/eslint-config-stripes": "^4.0.1",
+    "@folio/stripes": "^2.0.0",
```
Comment (aliaksei-chumakou) [inline package.json]: and here

File: package.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  "engines": {
+    "node": ">=8.9.4"
+  },
+  "stripes": {
+    "type": "plugin",
+    "pluginType": "find-interface",
+    "okapiInterfaces": {
+      "organizations-storage.interfaces": "1.0"
+    },
+    "displayName": "ui-plugin-find-interface.meta.pluginTitle"
```
Comment (aliaksei-chumakou) [inline package.json]: shouldn't it be `ui-plugin-find-interface.meta.title` ?

**Comment by aliaksei-chumakou** [comment]
@NikitaSedyx could you please add editorconfig also? thanks https://github.com/folio-org/ui-plugin-find-contact/blob/master/.editorconfig

File: package.json (hunk 2 above)
Comment (aliaksei-chumakou) [reply package.json]: https://github.com/folio-org/stripes/blob/master/doc/stripes-framework.md#upgrading-to-v26

File: package.json (hunk 4 above)
Comment (NikitaSedyx) [reply package.json]: yep

**Pull request merged**